### PR TITLE
updated code

### PR DESCRIPTION
**Pull request by asim516**

### CODE REVIEW

**Comment by CliqueAdmin** [comment]
Send a new pull request on training repository not the sandbox

**Review by CliqueAdmin — CHANGES_REQUESTED** [review]
Please move to training repository

File: src/test/java/playground/AppTest.java [hunk 1]
```diff
@@ -1,15 +1,65 @@
...
 
-  @Test
-  public void testAppHasAGreeting() {
-    App classUnderTest = new App();
-    assertNotNull("app should have a greeting",
-        classUnderTest.getGreeting());
+  }
+}
+
+class Rectangle
```
Comment (CliqueAdmin) [inline src/test/java/playground/AppTest.java:20]: Can you utilize inheritance here?

File: src/test/java/playground/AppTest.java [hunk 2]
```diff
@@ -1,15 +1,65 @@
-package playground;
+  package playground;
 
-import static org.junit.Assert.assertNotNull;
+class Shapes
+{
+  public static void main(String arg[])
+  {
+    Rectangle r = new Rectangle(10, 4);
```
Comment (CliqueAdmin) [inline src/test/java/playground/AppTest.java:7]: How can we use polymorphism here? Will that simplify it?

**Review by CliqueAdmin — CHANGES_REQUESTED** [review]

File: src/test/java/playground/AppTest.java (hunk 2 above)
Comment (asim516) [reply src/test/java/playground/AppTest.java:7]: yes i can will update the code again in sometime

**Pull request closed without merging**